src/EnergyPlus/HeatBalanceAirManager.cc

```diff
@@ -1698,11 +1772,11 @@ void GetSimpleAirModelInputs(EnergyPlusData &state, bool &ErrorsFound) // IF err
                 if (thisVentilation.MinIndoorTempSchedPtr > 0) {
                     if (Item1 == 1) {
                         if (!lNumericFieldBlanks(11))
-                            ShowWarningError(
-                                state,
-                                std::string{RoutineName} +
-                                    "The Minimum Indoor Temperature value and schedule are provided. The scheduled temperature will be used in the " +
-                                    cCurrentModuleObject + " object = " + cAlphaArgs(1));
+                            ShowWarningError(state,
+                                             std::string{RoutineName} +
```

Comment:
If you find yourself doing `std::string{RoutineName}`, turn around and use `format` for the whole thing.